app/Http/Controllers/Admin/AdminController.php

```diff
@@ -1,11 +1,35 @@
 <?php
+/**
+ * Parses and verifies the doc comments for files!
+ *
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @author   LoiTQ <loitq@lifull-tech.vn>
+ * @license  http://matrix.squiz.net/developer/tools/php_cs/licence BSD Licence
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
 
 namespace App\Http\Controllers\Admin;
 
 use App\Http\Controllers\Controller;
 use Illuminate\Http\Request;
+use Illuminate\Database\Eloquent\Model;
+use App\User;
 use Auth;
 
+/**
+ * Template Class Doc Comment
```

Comment:
'Template' 'Example' those word not a word should exist in Product

Reply:
Fixed, it deleted